java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -577,6 +578,40 @@ public static HostColumnVector fromStrings(String... values) {
     });
   }
 
+  /**
+   * Create a new string vector from the given values.  This API
+   * supports inline nulls.
+   */
+  public static HostColumnVector fromUTF8StringsBytes(byte[]... values) {
```

Comment:
I think this should be `fromUTF8Strings` since we already have methods like `appendUTF8String` that take `byte[]`.

Reply:
Thanks for review. Updated